util/unittests/test_process.py

```diff
@@ -158,7 +158,7 @@ def test01_is_process_running(self):
                                                         self.ssh_client),
                         'Testing true case for a valid process id. Remote')
 
-    def test02_is_process_running(self):
+    def test_is_process_running(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_is_process_running' from line 150

<!-- Comment by [SideCI](https://sideci.com) -->